### PR TITLE
feat(config): add undocumented parameter 6 to ZW3104

### LINKED ISSUE #2111: ZW3104 Smoothly Dim Config Option
I have a couple of these GE/Jasco ZW3104 dimmers and the experience is much improved with ZwaveJS. The only issue is that they instantly pop to the set dimming value instead of smoothly changing like an Inovelli dimmer switch. I read that there is an undocumented option that can change this behavior in newer firmwares, but I'm unsure how to edit my device configuration to manually add this option similarly as explained in the below article with OpenHAB.

https://community.openhab.org/t/jasco-ge-honeywell-z-wave-outlet-dimming-problem-resolution/109387

Does this link give enough information to implement this config option for the ZW3104 devices?

28166/28167/ZW3104 I believe are all the same device.

### PR DESCRIPTION
**Pull request by marcus-j-davies**
closes https://github.com/zwave-js/node-zwave-js/issues/2111

### CODE REVIEW

**Comment by marcus-j-davies** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0063/28166_zw3104.json [hunk 1]
```diff
@@ -59,6 +59,29 @@
 				}
 			]
 		},
+		"6": {
+			"$if": "firmwareVersion >= 5.21",
+			"label": "Light Level Change Mode",
+			"description": "Controls whether to fade to the next light level or jump straight there.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/28166_zw3104.json]:
```suggestion
			"label": "Smooth Level Changes",
```

File: packages/config/config/devices/0x0063/28166_zw3104.json [hunk 2]
```diff
@@ -59,6 +59,29 @@
...
+			"options": [
+				{
+					"label": "Smooth Level Changes",
+					"value": 1
+				},
+				{
+					"label": "Instant Level Changes",
+					"value": 0
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/28166_zw3104.json]:
```suggestion
			"options": [
				{
					"label": "Disabled",
					"value": 0
				},
				{
					"label": "Enabled",
					"value": 1
				}
			]
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Applies to both files

File: packages/config/config/devices/0x0063/28166_zw3104.json [hunk 3]
```diff
@@ -59,6 +59,28 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Enabled",
+					"value": 1
+				},
+				{
+					"label": "Disabled",
+					"value": 0
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/28166_zw3104.json]:
```suggestion
				{
					"label": "Disabled",
					"value": 0
				},
				{
					"label": "Enabled",
					"value": 1
				}
```

File: packages/config/config/devices/0x0063/ge_28167_zw3104.json [hunk 4]
```diff
@@ -41,6 +41,28 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Enabled",
+					"value": 1
+				},
+				{
+					"label": "Disabled",
+					"value": 0
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/ge_28167_zw3104.json]:
```suggestion
				{
					"label": "Disabled",
					"value": 0
				},
				{
					"label": "Enabled",
					"value": 1
				}
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
It was not an accident that I switched the options 😏

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**